aio_pika/abc.py

```diff
@@ -257,7 +265,17 @@ def __init__(
         arguments: Arguments,
         passive: bool = False,
     ):
-        raise NotImplementedError
+        raise NotImplementedError(
+            dict(
```

Comment:
Why do we need this data here? It's the only place where `NotImplementedError` is being raised with data..

Reply:
This code is a hack for linters it's never being called in the runtime